cli/pcluster/config/mappings.py

```diff
@@ -493,201 +591,244 @@
             ("template_url", {
                 # TODO add regex
                 "validators": [url_validator],
+                # Ignored during update since we force using previous template
+                "update_policy": UpdatePolicy.IGNORED
             }),
             ("shared_dir", {
                 "type": SharedDirParam,
                 "allowed_values": ALLOWED_VALUES["file_path"],
                 "cfn_param_mapping": "SharedDir",
                 "default": "/shared",
                 "validators": [shared_dir_validator],
+                "update_policy": UpdatePolicy.UNSUPPORTED
             }),
             ("enable_efa", {
                 "allowed_values": ["compute"],
                 "cfn_param_mapping": "EFA",
                 "validators": [efa_validator],
+                "update_policy": UpdatePolicy.UNSUPPORTED
             }),
             ("ephemeral_dir", {
                 "allowed_values": ALLOWED_VALUES["file_path"],
                 "default": "/scratch",
                 "cfn_param_mapping": "EphemeralDir",
+                "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("encrypted_ephemeral", {
                 "default": False,
                 "type": BoolParam,
                 "cfn_param_mapping": "EncryptedEphemeral",
+                "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("custom_ami", {
                 "cfn_param_mapping": "CustomAMI",
                 "allowed_values": ALLOWED_VALUES["ami_id"],
                 "validators": [ec2_ami_validator],
+                "update_policy": UpdatePolicy.UNSUPPORTED,
             }),
             ("pre_install", {
                 "cfn_param_mapping": "PreInstallScript",
                 # TODO add regex
                 "validators": [url_validator],
+                "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
             }),
             ("pre_install_args", {
                 "cfn_param_mapping": "PreInstallArgs",
+                "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
             }),
             ("post_install", {
                 "cfn_param_mapping": "PostInstallScript",
                 # TODO add regex
                 "validators": [url_validator],
+                "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
             }),
             ("post_install_args", {
                 "cfn_param_mapping": "PostInstallArgs",
+                "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
             }),
             ("extra_json", {
                 "type": ExtraJsonParam,
                 "cfn_param_mapping": "ExtraJson",
+                "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
             }),
             ("additional_cfn_template", {
                 "cfn_param_mapping": "AdditionalCfnTemplate",
                 # TODO add regex
                 "validators": [url_validator],
+                "update_policy": UpdatePolicy.UNSUPPORTED,
```

Comment:
Was set to UNSUPPORTED since the current approach is to always reuse the same cfn template, so ideally also substacks should be maintained unchanged. If really needed it could be updated using the ``--force`` flag. What's your opinion?